internal/api/v1/services.go

```diff
@@ -369,7 +369,12 @@ func (sc ServicesController) Delete(w http.ResponseWriter, r *http.Request) APIE
 		}
 	}
 
-	// Everything looks to be ok. Delete.
+	// Delete all service bindings. There can be service
+	// bindings whose apps are already deleted.
+	err = service.DeleteBindings(ctx)
```

Comment:
What happens if the app is __not__ deleted ? This looks as if it pulls the rug from under such an application.

Reply:
Removed this part of the code.